### PR TITLE
A few small functional test cleanups

### PR DESCRIPTION
**Pull request by phlogistonjohn**
Small quality-of-life improvements around the functional tests. It's pretty much just minor stuff that was annoying me and that I was hacking into the tests when I needed it.

### CODE REVIEW

File: tests/functional/TestManyBricksVolume/run.sh [hunk 1]
```diff
@@ -1,11 +1,8 @@
...
 
-CURRENT_DIR="$(pwd)"
-export HEKETI_SERVER_BUILD_DIR=../../..
-FUNCTIONAL_DIR="${CURRENT_DIR}/.."
-export HEKETI_SERVER="${FUNCTIONAL_DIR}/heketi-server"
+SCRIPT_DIR="$(cd "$(dirname "${0}")" && pwd)"
+FUNCTIONAL_DIR="${SCRIPT_DIR}/.."
 
-source "${FUNCTIONAL_DIR}/lib.sh"
+. "${FUNCTIONAL_DIR}/lib.sh"
```
Comment (obnoxxx) [inline tests/functional/TestManyBricksVolume/run.sh]: I am wondering why the change from "source" to "."
We use source throughout the code base.

Comment (phlogistonjohn) [reply tests/functional/TestManyBricksVolume/run.sh]: Well... I was trying to shorten those files. :-)
